frontend/catalyst/pennylane_extensions.py

```diff
@@ -115,19 +116,37 @@ class QFunc:
             the valid gate set for the quantum function
     """
 
-    def __init__(self, fn, device):
+    def __init__(self, fn, device):  # pragma: nocover
         self.func = fn
         self.device = device
         update_wrapper(self, fn)
 
+    @staticmethod
+    def _add_toml_file(device):
+        """Temporary function. This function adds the config field to devices.
+        TODO: Remove this function when `qml.Device`s are guaranteed to have their own
+        config file field."""
```

Comment:
Just checking, but is this the case for lightning devices now?

Reply:
It should be, but we are not running CI/CD with pennylane-lightning latest.

Reply:
It is incorrect, we not only need CI/CD to update pennylane lightning, but we also need the pennylane lightning package to be installed. The files are part of the python package, not of the C++ package. I think we should just keep the toml files in our repo for now and wait until the new pennylane lightning package is release via pip.

Reply:
Got it, yeah I was wondering if that was the case 👍